pkg/netpol/diff/diff.go

```diff
@@ -135,6 +137,22 @@ func (da *DiffAnalyzer) ConnDiffFromDirPaths(dirPath1, dirPath2 string) (Connect
 	return diffConnectionsLists(conns1Refined, conns2Refined, workloadsNames1, workloadsNames2)
 }
 
+func getPeersNamesFromPeersList(peers []eval.Peer) []string {
+	// create set from peers-strings
+	peersSet := make(map[string]bool, 0)
+	for _, peer := range peers {
+		peersSet[peer.String()] = true
+	}
+	// list of unique names
+	peersRes := make([]string, len(peersSet))
+	i := 0
+	for peerName := range peersSet {
+		peersRes[i] = peerName
+		i++
+	}
+	return peersRes
+}
+
```

Comment:
why not returning `map[string]bool`? (return `peersSet` instead of `peersRes`)?

Reply:
changed